package.json

```diff
@@ -6,11 +6,20 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "crowdin-clean": "rm -rf .crowdin && mkdir .crowdin",
+    "crowdin-import": "ts-node -O '{ \"module\": \"commonjs\" }' src/scripts/crowdin-import.ts",
+    "markdown-checker": "ts-node -O '{ \"module\": \"commonjs\" }' src/scripts/markdownChecker.ts",
+    "generate-heading-ids": "ts-node -O '{ \"module\": \"commonjs\" }' src/scripts/generateHeadingIds.ts",
+    "unused-translations": "ts-node -O '{ \"module\": \"commonjs\" }' src/scripts/unusedTranslations.ts",
+    "reorg-intl": "ts-node -O '{ \"module\": \"commonjs\" }' src/scripts/reorgIntlFiles.ts",
+    "optimize-images": "ts-node -O '{ \"module\": \"commonjs\" }' src/scripts/optimizeImages.ts",
```

Comment:
These were imported for completeness, but the underlying scripts are not yet imported. It doesn't break anything having them here, but we can also consider just removing them here, and bringing them in more formally in a separate PR.
```suggestion
```

Reply:
Personally, I'd prefer to declare them as we migrate/import each one. Mainly for organization and task tracking matters.